packages/lesmis/src/views/PageView.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { MuiThemeProvider } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import MuiContainer from '@material-ui/core/Container';
+import { Breadcrumbs, Toolbar } from '../components';
+import * as COLORS from '../constants';
+import { contentPageTheme } from '../theme';
+
+const TemplateBody = styled.section`
+  background: ${COLORS.GREY_F9};
+  padding-top: 3rem;
+  min-height: 800px; // fallback height for older browsers
+  min-height: 70vh;
+`;
+
+const TitleContainer = styled.section`
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  text-align: center;
+  border-bottom: 1px solid ${props => props.theme.palette.grey['400']};
+`;
+
+const Title = styled(Typography)`
+  font-weight: 600;
+  font-size: 1.75rem;
+  line-height: 2.6rem;
+`;
+
+export const PageView = ({ content }) => {
+  const { title, body, urlSegment } = content;
+  const breadcrumbs = [{ name: title, code: urlSegment }];
```

Comment:
Would remove the `code` here, I don't think it's used for anything? Will just need to add a condition check in the breadcrumbs component to not show an entity link if it's the last in the list _or_ has no code